lms/djangoapps/courseware/management/commands/dump_course_structure.py

```diff
@@ -95,6 +95,11 @@ def dump_module(module, destination=None, inherited=False, defaults=False):
 
     items = own_metadata(module)
 
+    # HACK: add discussion ids to list of items to export (AN-6696)
+    if module.location.category == 'discussion':
+        if 'discussion_id' not in items:
+            items['discussion_id'] = module.discussion_id
```

Comment:
Unless something truly horrible has gone wrong, in which case we'd probably want this to throw an error